### DISCUSSION TITLE
Using babel for web

### DISCUSSION THREAD

**Discussion by Zelfapp** [babel/babel#12373] [Q&A] [answered]
Just upgraded to core 3.7.0 from 2. V3 is adding `require` into the js output, which of course doesn't work on the web. I've searched and searched the docs and SO on this, but how do I refactor my `babel.config.js` here to remove the `require` directives so the js output is web compatible?

Incidentally I had to upgrade because I was getting a `regenerator` missing plugin error. You can see below that I've added that plugin and my code based on the browsers I'm targeting has several polyfills that are now being included, hence the `require` statements that are now showing up.

Currently, as a work around I'm using [browserify](https://github.com/browserify/browserify) to polyfill `require`.

```
module.exports = function getBabelConfiguration(api) {
    api.cache.never()
    return {
        presets: [
            [
                '@babel/preset-env',
                {
                    useBuiltIns: 'usage',
                    corejs: '3.7.0',
                    targets: {
                        edge: '14',
                        ie: '11',
                        firefox: '52',
                        chrome: '43',
                        safari: '9',
                    },
                },
            ],
        ],
        plugins: ['@babel/plugin-transform-regenerator'],
    }
}
```

**Comment by nicolo-ribaudo** [comment, marked as answer]
You can remove the `useBuiltIns` and `corejs` options from your config, and manually include with a `<script>` tag `core-js-bundle` and `regenerator-runtime/runtime.js` in your HTML.

**Reply by JackHowa** [reply]
Looking at this. I'm not sure if I'm following this as expected 

https://github.com/JackHowa/esmodule-no-module-doing-something/blob/webpack-minify-output/legacy-webpack.config.js 
https://github.com/JackHowa/esmodule-no-module-doing-something/blob/webpack-minify-output/index.html#L10 

![Screen Shot 2021-01-09 at 15 17 41](https://user-images.githubusercontent.com/5950956/104095261-d279dc00-528d-11eb-88c4-7f5378274d44.png)

**Reply by nicolo-ribaudo** [reply]
This question was about using it without webpack, since it asks how to avoid `require()`.
